src/components/LastKrappe/index.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react';
+import { Button } from 'react-bootstrap';
+
+class LastKrappe extends Component {
+    componentDidMount() {
+        this.setLastKrappe();
+    }
+
+    render() {
+        return (
+            <div>
+                <div>Jäbä krappes: { localStorage.getItem("krappe") }</div>
```

Comment:
I would probably reformat this.. Something like
<div>Jäbä krappes the last time: { localStorage.getItem("krappe") || 'NEVER' } </div>